### PR TITLE
Adding GUI

### PR DESCRIPTION
**Pull request by michbad**

### CODE REVIEW

File: panoptisong [hunk 1]
```diff
@@ -2,12 +2,33 @@
...
+    params="$cur_dir/${prefix}_parameters"
+    birds="$cur_dir/${prefix}_birds"
+fi
+
+source $params
+
+global_attributes="-a room=$room -a species=$species -a lightson=$lightson -a lightsoff=$lightsoff -a jackserver=$servername"
 jdetect_params="--close-period $closeperiod --close-rate $closerate --close-thresh $closethresh --open-thresh $openthresh --open-rate $openrate"
 
+if [ "$prefix" == "" ]; then
```
Comment (kylerbrown) [inline panoptisong]: why isn't this included with line 10?

File: panoptisong [hunk 2]
```diff
@@ -2,12 +2,33 @@
 
 # move to source directory
 cd "$(dirname "$0")" || exit
+cur_dir=$(pwd)
 ## load parameters
-source ./parameters
 
-global_attributes="-a experimenter=$experimenter -a room=$room -a species=$species -a lightson=$lightson -a lightsoff=$lightsoff -a jackserver=$servername"
+prefix="$1" #which set of parameters we are loading
```
Comment (kylerbrown) [inline panoptisong:8]: what's going on here? what's $1?

File: panoptisong [hunk 3]
```diff
@@ -2,12 +2,33 @@
...
+global_attributes="-a room=$room -a species=$species -a lightson=$lightson -a lightsoff=$lightsoff -a jackserver=$servername"
 jdetect_params="--close-period $closeperiod --close-rate $closerate --close-thresh $closethresh --open-thresh $openthresh --open-rate $openrate"
 
+if [ "$prefix" == "" ]; then
+    bird_file="$birds"
+fi
+
+
+echo $params
+echo $birds
```
Comment (kylerbrown) [inline panoptisong]: why?

File: panoptisong [hunk 4]
```diff
@@ -2,12 +2,33 @@
...
+bird_file="$birds"
```
Comment (kylerbrown) [inline panoptisong]: is this variable used?

File: panoptisong [hunk 5]
```diff
@@ -29,14 +50,18 @@ function record_bird {
     shift
     local box="$1"
     shift
+    local experimenter="$1"
+    shift
+    # trig_channel must be the last field in the file, because it's used both in trig_channel
```
Comment (kylerbrown) [inline panoptisong]: comment does not match logic, what's going on here?

File: README.md [hunk 6]
```diff
@@ -11,15 +11,28 @@ Clone this repository: `git clone https://github.com/kylerbrown/panoptisong.git`
...
+You can make use of the profiles even when running panoptisong from the command line, by typing
+`./panoptisong my_config` (but you can use the default files with just `./panoptisong`)
+
 birds
 ------
-Edit the file `birds` to include your birds and their microphone channels. Rows starting with # are ignored. The three positional variables are BIRD BOX CHANNEL. For example, the line 
+Edit the file `birds` to include your birds and their microphone channels. Rows starting with # are ignored. The four positional variables are BIRD BOX EXPERIMENTER CHANNEL. The values are separated with tabs. For example, the line 
 
-    bk196 1 system:capture_3
+    bk196 1 jack system:capture_3
```
Comment (kylerbrown) [inline README.md]: `jack` seems like a poor choice of name here.

File: panoptisong (hunk 2 above)
Comment (michbad) [inline panoptisong:8]: $1 is the optional second parameter, that allows you to use "profiles" of parameter/bird files. If it's not present, then we just use the standard birds and parameters files. I've documented that.

**Comment by michbad** [comment]
I think it should be good now.

**Comment by kylerbrown** [comment]
thanks!

**Pull request merged**